drivers/spi/spi_ll_stm32.c

```diff
@@ -479,22 +499,23 @@ static void spi_stm32_complete(const struct device *dev, int status)
 	}
 #endif
 
-	if (LL_SPI_GetMode(spi) == LL_SPI_MODE_MASTER) {
-		while (ll_func_spi_is_busy(spi)) {
-			/* NOP */
-		}
-
-		spi_stm32_cs_control(dev, false);
```

Comment:
Are you reverting https://github.com/zephyrproject-rtos/zephyr/pull/66381?
If, yes, please do it in a dedicated commit with a proper justification

Reply:
It's not a reversion: I have moved it from where it was to below because now the "way of waiting for the SPI to finish" has 2 forms: one with a while loop for when interrupts are not enabled, and another one with an `if` for when interrupts are enabled. In any case, CS needs to be de-asserted after the driver has waited enough, so (to avoid replicating code) this statement needs to be moved to after the wait operation. But it's equivalent in terms of functionality.

Reply:
To avoid confusion a dedicated commit would be welcome